src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Windows.NetStandard.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if WINDOWS || WINDOWS_UWP || NET472_OR_GREATER || NETSTANDARD2_0
+﻿#if WINDOWS || WINDOWS_UWP || NET472_OR_GREATER || NET7_0
```

Comment:
I'm thinking this should be:

```suggestion
﻿#if WINDOWS || WINDOWS_UWP || UNO_REFERENCE_API
```